modules/container-base/README.md

```diff
@@ -0,0 +1,56 @@
+# Dataverse Base Container Image
```

Comment:
I'm surprised that it reads "Dataverse Base Container Image" here but "Application Base Image" on the [rendered version](https://dataverse-guide--8933.org.readthedocs.build/en/8933/container/base-image.html). But as far as the code here is concerned, shouldn't it be "Dataverse Container Base Image"?

Reply:
Beware - this text is not rendered within the guide, but at https://hub.docker.com/r/gdcc/base. I would like the Hub page to be much shorter and self explaining. Do you think we should align both?

Reply:
I thought the last link you provided was a more reader-friendly version of the code. But then I saw several sections weren't on GitHub. Either way, I was just curious about the phrasing of this specific section. Nothing vital here  :thumbsup:

Reply:
OK closing then. Thanks again! :smile: